### ISSUE TITLE
Downloading a publicly shared calendar via the Download Button fails

### ISSUE THREAD

**Issue by founderio** [nextcloud/server#9806]
### Steps to reproduce
1. Share a calendar via public link
2. Open the public link
(Example link: https://cloud.dhelixnet.de/index.php/apps/calendar/p/NIH051N8XT4K60IS/Kurse)
3. Clickt the "Download" button on the top right
(Example: https://cloud.dhelixnet.de/remote.php/dav/public-calendars/NIH051N8XT4K60IS?export)

### Expected behaviour
The download of the file in ics format should start.

### Actual behaviour
I see a full screen message: `This is the WebDAV interface. It can only be accessed by WebDAV clients such as the Nextcloud desktop sync client.`

### Server configuration

**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MariaDB

**PHP version:**
7.0

**Nextcloud version:** (see Nextcloud admin page)
13.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated all the way from an old ownCloud instance

**Where did you install Nextcloud from:**
Manual download

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.6.1
  - admin_audit: 1.3.0
  - bruteforcesettings: 1.0.3
  - calendar: 1.6.1
  - checksum: 0.4.0
  - comments: 1.3.0
  - contacts: 2.1.5
  - dav: 1.4.7
  - deck: 0.3.1
  - external: 3.0.3
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_downloadactivity: 1.2.0
  - files_external: 1.4.1
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.2.1
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.1
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - socialsharing_email: 1.0.3
  - spreed: 3.2.2
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.5
  - twofactor_backupcodes: 1.2.3
  - twofactor_totp: 1.4.1
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0
Disabled:
  - encryption
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "maintenance": false,
        "version": "13.0.3.2",
        "installedat": "1335529785.0068",
        "lastupdatedat": "1342630087.1883",
        "upgrade.disable-web": true,
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "trusted_domains": [
            "cloud.dhelixnet.de"
        ],
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "mail_smtpsecure": "tls",
        "versions": "true",
        "enable_previews": true,
        "preview_max_scale_factor": 10,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/cloud.dhelixnet.de",
        "check_for_working_webdav": true,
        "check_for_working_wellknown_setup": true,
        "check_for_working_htaccess": true,
        "loglevel": "1",
        "log_type": "file",
        "logfile": "\/var\/log\/nextcloud\/nextcloud.log",
        "log_rotate_size": 104857600,
        "updater.release.channel": "stable"
    }
}

```
</details>

**Are you using external storage, if yes which one:** no

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Chromium

**Operating system:** Linux

#### Web server error log
<details>
<summary>Web server error log</summary>
No Error log entries, only access log
```
[IP] - - [08/Jun/2018:19:08:21 +0000] "GET /remote.php/dav/public-calendars/NIH051N8XT4K60IS?export HTTP/1.1" 200 880 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.139 Safari/537.36"
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
{"reqId":"3S5tFRCvHrJvzy2HijJk","level":4,"time":"2018-06-08T19:04:03+00:00","remoteAddr":"2001:16b8:2843:3a00:8981:2996:5f01:936","user":"admin","app":"webdav","method":"REPORT","url":"\/remote.php\/dav\/public-calendars\/NIH051N8XT4K60IS\/","message":"Exception: {\"Exception\":\"Sabre\\\\DAV\\\\Exception\\\\ReportNotSupported\",\"Message\":\"\",\"Code\":0,\"Trace\":\"#0 [internal function]: Sabre\\\\DAV\\\\CorePlugin->httpReport(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#1 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(479): Sabre\\\\Event\\\\EventEmitter->emit('method:REPORT', Array)\\n#3 \\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(254): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#4 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/lib\\\/Server.php(287): Sabre\\\\DAV\\\\Server->exec()\\n#5 \\\/var\\\/www\\\/owncloud\\\/apps\\\/dav\\\/appinfo\\\/v2\\\/remote.php(35): OCA\\\\DAV\\\\Server->exec()\\n#6 \\\/var\\\/www\\\/owncloud\\\/remote.php(164): require_once('\\\/var\\\/www\\\/ownclo...')\\n#7 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/CorePlugin.php\",\"Line\":725}","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/66.0.3359.139 Safari\/537.36","version":"13.0.3.2"}
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Navigated to https://cloud.dhelixnet.de/remote.php/dav/public-calendars/NIH051N8XT4K60IS?export
NIH051N8XT4K60IS:1 Refused to load the image 'https://cloud.dhelixnet.de/favicon.ico' because it violates the following Content Security Policy directive: "default-src 'none'". Note that 'img-src' was not explicitly set, so 'default-src' is used as a fallback.
```
</details>

**Comment by MorrisJobke** [comment]
Most likely fixed by #9798 in 13.0.4 (released today)

**Comment by founderio** [comment]
Yup, fixed!
